gadget/layout.go

```diff
@@ -25,91 +25,91 @@ import (
 	"sort"
 )
 
-// PositioningConstraints defines the constraints for positioning structures
-// within a volume
-type PositioningConstraints struct {
+// LayoutConstraints defines the constraints for arranging structures within a
+// volume
+type LayoutConstraints struct {
 	// NonMBRStartOffset is the default start offset of non-MBR structure in
 	// the volume.
 	NonMBRStartOffset Size
 	// SectorSize is the size of the sector to be used for calculations
 	SectorSize Size
 }
 
-// PositionedVolume defines the size of a volume and positions of all the
+// LaidOutVolume defines the size of a volume and arrangement of all the
 // structures within it
-type PositionedVolume struct {
+type LaidOutVolume struct {
 	*Volume
 	// Size is the total size of the volume
 	Size Size
 	// SectorSize sector size of the volume
 	SectorSize Size
-	// PositionedStructure are sorted in order of 'appearance' in the volume
-	PositionedStructure []PositionedStructure
+	// StructureLayout are the structures within the volume, sorted by their
+	// start offsets
+	StructureLayout []LaidOutStructure
 	// RootDir is the root directory for volume data
 	RootDir string
 }
 
-// PositionedStructure describes a VolumeStructure that has been positioned
-// within the volume
-type PositionedStructure struct {
+// LaidOutStructure describes a VolumeStructure that has been placed within the
+// volume
+type LaidOutStructure struct {
 	*VolumeStructure
 	// StartOffset defines the start offset of the structure within the
 	// enclosing volume
 	StartOffset Size
-	// PositionedOffsetWrite is the resolved position of offset-write for
-	// this structure element within the enclosing volume
-	PositionedOffsetWrite *Size
+	// AbsoluteOffsetWrite is the resolved absolute position of offset-write
+	// for this structure element within the enclosing volume
+	AbsoluteOffsetWrite *Size
 	// Index of the structure definition in gadget YAML
 	Index int
 
-	// PositionedContent is a list of raw content included in this structure
-	PositionedContent []PositionedContent
+	// ContentLayout is a list of raw content included in this structure
+	ContentLayout []LaidOutContent
```

Comment:
Again, per pattern this would have been LaidOutContent, no?  could just be Content though.

Don't know if there are more similar cases?

Reply:
Per IRC discussion, renamed to LaidOutContent